### DISCUSSION TITLE
Need Help with Github Student Developer Pack Verification Process

### DISCUSSION THREAD

**Discussion by ahnayef** [community/community#60009] [GitHub Education]
### Hi everyone,

I wanted to share a little issue I'm facing with the Github Student Developer Pack. I've been enjoying it for around 6-7 months now, and it's been a fantastic resource for me. However, I'm having some trouble with the verification process, and I could really use your help.

This will be my third time verifying myself as a student, and in the past, it hasn't been a problem. Each verification usually lasts for 2-3 months, which is great. However, my university doesn't provide us with email addresses, which is a bit inconvenient. So, I've been using my personal email as the primary one on my GitHub account.

Previously, I could verify my student status using official documents, but this time it's not letting me proceed with my personal email. I've searched for an option to verify using documents, but unfortunately, I haven't found one. The message I received says:

**We require applicants of \<university name\> to use one of these school-issued email addresses to apply:
\- <university website\>
Please select a different email address.**

I'm a bit stuck and not sure what to do next. Has anyone else faced a similar situation? If you have any advice or suggestions, I would greatly appreciate it.

Thank you so much for your time and support.

**Comment by UnbearableLightnessOfBeing** [comment]
Same issue

**Reply by ahnayef** [reply]
It's been 3 weeks but no response.

**Comment by IgnacioPalma** [comment]
@samus-aran

**Reply by samus-aran** [reply]
Thanks for the mention @IgnacioPalma ❤️

**Comment by samus-aran** [comment]
Hi @ahnayef 👋 

First off I want to thank you for sharing your experiences and that we absolutely understand your frustration. We are trying our best to continually improve our system to be as easy as possible for legitimate students everywhere to verify their academic status, while also ensuring that bad actors impersonating students don't jeopardise our ability to provide academic benefits of value. 

Occasionally it is necessary for us to ask you to reverify your current academic status in order to continue receiving your GitHub Education benefits. We may require this when seemingly-legitimate applicants have submitted academic affiliation proof which has been discovered to be used by bad actors as well.

As you can imagine there are many types of academic verification and our system is constantly evolving to accommodate this without setting the bar too high. However sometimes we may require a little more evidence from the student applying and we thank you for being patient. 

In this case it would be the Academic email(s) which has been associated with this school previously.

If you believe your applied school's details are incorrect please raise a ticket with [Education support ](https://support.github.com/contact/education)using the dropdown "My selected school has incorrect or incomplete information e.g. domains or campus location". 👍 Please provide as much evidence as possible to help verify the details that require updating. All applications must go through [GitHub Global Campus Student Application form](https://education.github.com/discount_requests/student_application). We are unable to provide manual reviews or upload links for individual applicants but we can review if the school details are accurate.

**Reply by ahnayef** [reply]
Hi @samus-aran  👋

Thank you for your response and understanding my situation. I truly appreciate the efforts you are making to ensure legitimate students can access the GitHub Student Developer Pack.

Regarding the verification process, I would like to clarify that my university does not provide email addresses to its students. As a result, I have been using my personal email address on GitHub, which has previously allowed me to verify my student status successfully.

One thing to note that there isn't any problem with university's details. It's even listed on Student application form drop down list.

However, I do have all the official documents and proof of enrollment to verify my academic status. I am more than willing to provide any necessary documents to support my claim as a student.

Considering the unique circumstance of my university not issuing email addresses, I kindly request if there are alternative methods available for verifying my student status. I am fully committed to cooperating and providing any additional information that may be required.

Once again, thank you for your attention to this matter. I look forward to resolving the verification issue and continuing to benefit from the GitHub Student Developer Pack.

**Comment by SuperBueno1** [comment]
Hi,
I'm having trouble too. Verification keeps failing, I tried the multifactor, not using vpn, the way it makes you take a pic is really odd, it's challenging but I took a pic showing proof of enrollment. I kind of feel like if I could just email the proof it could be easier. I'm trying to go through this so I can use copilot with vsCode. It's ironic, I have copilot and can use the web interface for free since it recognized my student email but I can't get this process down for GH.
